### DISCUSSION TITLE
Exposing `sndspeed` cvar in sound menu?

### DISCUSSION THREAD

**Discussion by vide0hanz** [Novum/vkQuake#658] [Ideas]
Since we now have new options in the menus which play nice with the rerelease pak0.pak I'm wondering if its a good idea to add an option to switch between classic/remastered sounds as well?

A simple way to do it would be to leave "classic" at the default 11025 sample rate, and "remastered" could be set to something like 44100 to take advantage of the higher quality source files contained within the rerelease pakfiles. It would coexist nicely with the classic/remastered models option, for example.

I'm not sure if this is a good idea or not since some mods might not include audio sources that would benefit from being played back at a higher sample rate, which may lead to some confusion. The same can also be said about the models menu option too... 

On that note, perhaps its worth updating the readme to recommend that vkquake be used with the rerelase basedir now for the best experience, since the original pak files are unable to take advantage of features like these.

EDIT: Just noticed issue #480 has more in-depth discussion about this, seems this is more nuanced than I initially thought.

**Comment by Novum** [comment]
Yeah I wasn't comfortable about this because I don't know enough about the sound system. I definitely thought about it.
